packages/clerk-js/src/utils/instance.test.ts

```diff
@@ -11,10 +11,14 @@ describe('validateFrontendApi(str)', () => {
     ['clerk.abcef.12345.lcl', true],
     ['clerk.abcef.12345.lcl.dev', true],
     ['clerk.abcef.12345.stg.dev', true],
+    ['clerk.ab-cef.12-345.lcl.dev', true],
+    ['clerk.ab-cef.123-45.stg.dev', true],
     ['clerk.abcef.12345.lclstage.dev', true],
     ['clerk.abcef.12345.stgstage.dev', true],
     ['clerk.abcef.12345.dev.lclclerk.com', true],
+    ['clerk.ab-cef.123-45.dev.lclclerk.com', true],
     ['clerk.abcef.12345.stg.lclclerk.com', true],
+    ['clerk.ab-cef.123-45.stg.lclclerk.com', true],
```

Comment:
A more realistic set of valid hostname examples:

```
clerk.happy.hippo-1.lcl.dev
clerk.sad.panda-99.stg.dev
clerk.foo.bar-12.dev.lclclerk.com
clerk.foo.bar-12.stg.lclclerk.com
```

Reply:
Great, thanks for the help Agi!